config/mempool.mk

```diff
@@ -21,7 +21,14 @@ num_cores_per_tile ?= 4
 banking_factor ?= 4
 
 # Radix for hierarchical AXI interconnect
-axi_hier_radix ?= 20
+axi_hier_radix ?= 17
 
 # Number of AXI masters per group
 axi_masters_per_group ?= 1
+
+# Number of DMA backends in each group
+dmas_per_group ?= 1 # Brust Length = 16
+
+# L2 Banks/Channels
+l2_size  ?= 4194304  # 400000
+l2_banks ?= 4
```

Comment:
Missing newline

Reply:
Hi Sam, the EOF line will break our CI, which is the same reason for non-EOF in our current configuration files.

Reply:
Oh, what? Ok, let's look into this in a separate PR to not postpone the DRAM further.